packages/location-management/src/components/LocationUnitAddEdit/index.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router';
+import { getAccessToken } from '@onaio/session-reducer';
+import { Ripple } from '@onaio/loaders';
+import { OpenSRPService } from '@opensrp/server-service';
+import {
+  LOCATION_UNIT_FINDBYPROPERTIES,
+  LOCATION_HIERARCHY,
+  LOCATION_UNIT_GROUP_ALL,
+} from '../../constants';
+import { API_BASE_URL } from '../../configs/env';
```

Comment:
Let's retrieve the base URL from props

Reply:
@haseemVD have created a separate issue to deal with it .. it will be solved separately

Reply:
Created separate [issue](https://github.com/OpenSRP/web/issues/256) for this